templates/status.html

```diff
@@ -66,6 +69,27 @@
             var signup= "{{signup}}".split(" ");
             var logos = JSON.parse({{ logos|tojson|safe }});
 
+            var real_name = {{real_name}}
+            // Modal for telling user to not to use real names
+            if(real_name == 'yes'){
+                // Create Modal
+                swal({
+                    icon: "warning",
+                    title: "Wait",
+                    text: "Seems like you are trying to use a (full) real name as a username! It's never a good idea! Think of some other username.",
+                    button: "Got it! Try a new username!",
+                    closeOnEsc: false,
+                    closeOnClickOutside: false,
+                    width: '800px'
```

Comment:
You're mixing double and single quotes in this block. Try to choose one and stick with it.

Reply:
I see the width parameter doesn't do anything. So removing it.

Too many SweetAlerts around on the internet.